app/parsers/hyku_addons/csv_parser.rb

```diff
@@ -45,17 +45,17 @@ def retrieve_cloud_files(files)
         target_file = File.join(files_path, file["file_name"])
         # Now because we want the files in place before the importer runs
         # Problematic for a large upload
-        Bulkrax::DownloadCloudFileJob.perform_now(file, target_file)
+        ::Bulkrax::DownloadCloudFileJob.perform_now(file, target_file)
       end
       nil
     end
 
     def entry_class
-      HykuAddons::CsvEntry
+      CsvEntry
```

Comment:
I know this strictly doesn't need `HykuAddons::` prepended, but since there is a Bulkrax class of exactly the same name it really helps the reader understand which one is being referred to.  Same with the `admin_set_entry_class`.

Reply:
I do agree that understanding of the code is paramount, but if its namespaced `HykuAddons` within a `HykuAddons` module, then it seems like a smell. What do you think? Is it better to present "proper" code and reduces as much redundancy as possible, or to explain as much as is possible to the reader?